src/helperFunctions/helperFunctions.ts

```diff
@@ -148,6 +149,23 @@ const extractFormData = (stepData: Step, body: RequestBody): StepValue => {
     return body[stepData.id] as StepValue;
   }
 
+  if (stepData.id === "data-type") {
+    console.log(stepData, body)
+    return {
+      "personal": {
+        checked: body["data-type"]?.includes("personal"),
+      },
+      "special": {
+        checked: body["data-type"]?.includes("special"),
+      },
+      "none": {
+        explanation: body["none"],
```

Comment:
in the prototype I think there's explanation textarea. Looks like is just the checkbox: https://dds-e2e.herokuapp.com/request/030-what-type-data-cat